kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -567,35 +567,27 @@ def _should_zk(self, zk_hosts_ports, interval, kafka_collect=False):
 
         return should_zk
 
-    @staticmethod
-    def _read_config(instance, key, cast=None):
-        val = instance.get(key)
-        if val is None:
-            raise BadKafkaConsumerConfiguration('Must provide `%s` value in instance config' % key)
```

Comment:
note that this validation logic is already happening in the callers, so it does not need to be added anywhere when removing this method.